### PR TITLE
[es] Localization scalability.md

### PR DESCRIPTION
**Pull request by ramrodo**
### Describe your changes

Localization of `scalability.md`

### Related issue number or link (ex: `resolves #issue-number`)

resolves https://github.com/cncf/glossary/issues/1402

### Checklist before opening this PR (put `x` in the checkboxes)
- [X] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [X] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

File: content/es/scalability.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
+title: Escalabilidad
+status: Completed
+category: Propiedad
+tags: ["fundamental", "property", ""]
+---
+
+La escalabilidad se refiere a qué tan bien puede crecer un sistema.
```
Comment (electrocucaracha) [inline content/es/scalability.md:8]: Hasta donde [se](https://www.ejemplos.co/que-o-que/) lleva acento cuando se utiliza en preguntas.
```suggestion
La escalabilidad se refiere a que tan bien puede crecer un sistema.
```

File: content/es/scalability.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+Eso está aumentando la capacidad de hacer lo que se supone que debe hacer el sistema.
```
Comment (electrocucaracha) [inline content/es/scalability.md:9]: Me parece que esto debería llevar una coma,
```suggestion
Esto es, aumentando la capacidad para hacer lo que sea que el sistema deba hacer.
```

File: content/es/scalability.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+Por ejemplo, un [clúster](/es/cluster/) de [Kubernetes](/es/kubernetes/) escala
+aumentando o reduciendo el número de aplicaciones [en contenedores](/es/containerización/),
```
Comment (electrocucaracha) [inline content/es/scalability.md:11]:
```suggestion
aumentando o reduciendo el número de aplicaciones [en contenedores](/es/containerization/),
```

File: content/es/scalability.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+pero esa escalabilidad depende de varios factores.
+¿Cuántos [nodos](/es/nodes/) tiene, cuántos [contenedores](/container/) puede manejar cada nodo,
```
Comment (electrocucaracha) [inline content/es/scalability.md:13]:
```suggestion
¿Cuántos [nodos](/es/nodes/) tiene?, ¿cuántos [contenedores](/container/) puede manejar cada nodo?,
```

**Review by electrocucaracha — CHANGES_REQUESTED** [review]

**Review by electrocucaracha — APPROVED** [review]

**Comment by ramrodo** [comment]
@krol3 @92nqb could you review this PR, please? Thanks :)

**Review by krol3 — APPROVED** [review]
LGTM

**Pull request merged**